flask_rebar/validation.py

```diff
@@ -115,7 +115,13 @@ def disallow_extra_fields(self, processed_data, original_data):
     RequestSchema = Schema
 
 
+# Define a constructor to please some IDE type checking
 class ResponseSchema(ActuallyRequireOnDumpMixin, Schema):
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+
+
+class StrictSchema(RequestSchema, ResponseSchema):
     pass
```

Comment:
Missing docstring for public class.

Once added, the `pass` will be unnecessary.